templates/rhsm-katello-reconfigure.erb

```diff
@@ -83,6 +83,13 @@ else
       --server.port="$PORT" \
       --rhsm.repo_ca_cert="%(ca_cert_dir)s$KATELLO_SERVER_CA_CERT" \
       --rhsm.baseurl="$BASEURL"
+
+    # Older version of subscription manager may not recognize
```

Comment:
version should be versions

Reply:
1 nitpick from me but other than that ack.

Reply:
updated